CinemaTicketSalesSystem/ApplicationDbMovies/ApplicationDbMovies/Configurations/MovieActorConfig.cs

```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Data.Entity.ModelConfiguration;
-using System.Data.Entity;
```

Comment:
Commit is called 'Minor bug fix' but it also has usings cleanup.
In future name you commits so they explicitly tell what is going on in the commit.